src/index.html

```diff
@@ -3,9 +3,47 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css" /></>
     <title>Wbpack Exercise</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <h1>To do list task</h1>
+    <div class="todo-container">
+        <div>            
+            <div class="todo-header">
+                <span class="describtion">Today's To Do</span>
+                <span class="more"><i class="bi-arrow-clockwise"></i></span>
+            </div>
+            <div class="todo-entry">                    
+                <span class="describtion">
+                    <form class="form" onsubmit="event.preventDefault()">
+                        <input class="enter-todo" type="text" placeholder="Add you List..." />
+                    </form>                        
+                </span>
+                <span class="more"><i class="bi-arrow-return-left"></i></span>
+            </div>
+            <div class="todo-list">
+                <!-- <div>
+                    <span class="desc">
+                        <input type="checkbox"/>
+                        <span>My list-1</span>
+                    </span>
+                    <span class="more">
+                        M
+                    </span>
+                </div>
+                <div>
+                    <span class="desc">
+                        <input type="checkbox" id="1"/>
+                        <span>My list-1</span>
+                    </span>
+                    <span class="more">
+                        M
+                    </span>
+                </div> -->
```

Comment:
- Please remove this comment. Comments make our code look ugly.

Reply:
Okay, I will remove it, i just use it for checking the layout of JS added elements.